tests/e2e/specs/MochaHooks.ts

```diff
@@ -63,7 +63,7 @@ exports.mochaHooks = {
             }
         },
         async function stopTheDriver(): Promise<void> {
-            if (!TestConstants.TS_DEBUG_MODE) {
+            if (!TestConstants.TS_DEBUG_MODE && TestConstants.USE_CHROME_DRIVER_FOR_TEST) {
```

Comment:
I can't find "USE_CHROME_DRIVER_FOR_TEST" constant in TestConstans.ts file.

Also, does the name of driver matter in this case?
Can it be just `USE_WEB_DRIVER_FOR_TEST` name (given that we could theoretically run the test using ChromeDriver, SafariDriver etc.)?